src/components/fullHeight/index.tsx

```diff
@@ -0,0 +1,9 @@
+import React, { FC, PropsWithChildren } from 'react';
+import { Box } from '@chakra-ui/react';
+
+const FullHeight: FC<PropsWithChildren> = ({ children }) => (
+  <Box minHeight="100vh">{children}</Box>
```

Comment:
Do you think it makes sense to add that value in our sizing tokens?

Reply:
I will not move it for now as I don't expect us to have other cases where we need the full device height besides this component. Let me know if you tink differently and I can move it in another PR 🙂